src/_pytest/python.py

```diff
@@ -1508,18 +1508,17 @@ def _showfixtures_main(config: Config, session: Session) -> None:
                 tw.line()
                 tw.sep("-", f"fixtures defined from {module}")
                 currentmodule = module
-        if verbose <= 0 and argname[0] == "_":
+        if verbose <= 0 and argname.startswith("_"):
```

Comment:
Seems like an unrelated and potentially unneeded change? (but I'm okay with both variants).

Reply:
I wanted to keep this consistent, noticed a similar check taking the `startswith` form. So made this change.